packages/react-instantsearch-dom/src/lib/voiceSearchHelper/index.ts

```diff
@@ -1,5 +1,24 @@
 // copied from https://github.com/algolia/instantsearch.js/blob/688e36a67bb4c63d008d8abc02257a7b7c04e513/src/lib/voiceSearchHelper/index.ts
 
+// begin - wrong SpeechRecognition-related types
```

Comment:
```suggestion
// #region wrong SpeechRecognition-related types
```

Reply:
this should allow it to collapse, but not 100% sure

Reply:
Yup, it works!